apps/dev-portal-web/domains/docs/widgets/AmountDistributionCalculator.tsx

```diff
@@ -0,0 +1,352 @@
+import { Col, Row, RowProps } from 'antd'
+import React, { CSSProperties, useCallback, useEffect, useState } from 'react'
+import { useIntl } from 'react-intl'
+
+import { split, createRecipientKey } from '@open-condo/billing/tools/billingCentrifuge'
+import type { TSplit, TDistributionItem, TSplitOptions } from '@open-condo/billing/tools/billingCentrifuge'
+import { Plus, Trash } from '@open-condo/icons'
+import { Alert, Button, Card, Input, List, MarkdownCodeWrapper, Switch, Tabs, Typography } from '@open-condo/ui'
+
+const CARD_PADDING = 8
+const GUTTER_COMPACT_ROW: RowProps['gutter'] = [8, 8]
+const GUTTER_ROW: RowProps['gutter'] = [16, 16]
+const RECIPIENT_CHECKBOX_ROW_STYLE: CSSProperties = { height: '28px' }
```

Comment:
Consider using css modules instead. We don't hard-code style in here

Reply:
Ok, moved to module